src/Adapter/LegacyContext.php

```diff
@@ -228,17 +236,37 @@ public function getEmployeeLanguageIso()
     }
 
     /**
-     * Returns Currency set for the current employee.
+     * Returns Currency set for the current employee, you can force to return the default
+     * currency if the employee one is not set.
+     *
+     * @param bool $defaultFallback
+     *
+     * @return Currency
      */
-    public function getEmployeeCurrency()
+    public function getEmployeeCurrency($defaultFallback = false)
```

Comment:
Starting with PHP 7.2, modifying signature of methods leads to PHP notices when an override does not match. Also, @mickaelandrieu previously reported that this kind of parameter are not intuitively understandable.

Can't you keep the current behavior and call this method with a ternary condition?

```php
$currency = $this->getEmployeeCurrency() ?: $this->getDefaultCurrency();
```

Reply:
ok you're right I will remove this modification to avoid BC breaks

Reply:
done